moesif/plugins/log.lua

```diff
@@ -5,7 +5,7 @@ local _M = {}
 -- Moesif Context
 local ctx = {
     application_id = nil,
-    batch_size = 25,
```

Comment:
lets make it 5 for now? obviously if its a customer with large volume, we ll work with them closely and adjust batch_size to higher value.

Reply:
okay, we've default to 5 now.